test/mocking_helper.py

```diff
@@ -0,0 +1,67 @@
+import sys
+from importlib import util
+from types import NoneType
```

Comment:
We're still bound to Python 3.9+, so `NoneType` is [unfortunately unavailable](https://docs.python.org/3.10/library/types.html#types.NoneType). `None` should already be enough, though